LEGO1/legovehiclebuildstate.h

```diff
@@ -0,0 +1,61 @@
+#ifndef LEGOVEHICLEBUILDSTATE_H
+#define LEGOVEHICLEBUILDSTATE_H
+
+#include "legostate.h"
+#include "mxstring.h"
+#include "decomp.h"
+
+#include "decomp.h"
+
+// VTABLE 0x100d66e0
+// SIZE 0x50 (from 1000acd7)
+class LegoVehicleBuildState : public LegoState
+{
+public:
+  LegoVehicleBuildState(char* p_classType);
+
+  // OFFSET: LEGO1 0x10025ff0
+  inline virtual const char *ClassName() const override // vtable+0x0c
+  {
+    return this->m_className.GetData();
+  }
+
+  // OFFSET: LEGO1 0x10026000
+  inline virtual MxBool IsA(const char *p_name) const override // vtable+0x10
+  {
+    return !strcmp(p_name, this->m_className.GetData()) || LegoState::IsA(p_name);
+  }
+
+public:
+  struct UnkStruct
+  {
+    undefined4 m_unk0;
```

Comment:
Could you specify the members' offsets instead of numbering them sequentially? Thank you!